### ISSUE TITLE
Chrome for iOS closeOnVerticalDrag bug

### ISSUE THREAD

**Issue by hudochenkov** [dimsemenov/PhotoSwipe#985]
In Chrome for iOS with closeOnVerticalDrag enabled dragging image to the very top is freezes image in that position. But after any interaction with the page image goes where it should be.

On Safari and Chrome for Android this bug is not reproduced.

![img_0721](https://cloud.githubusercontent.com/assets/654597/10275933/3a4737c8-6b54-11e5-87a3-35fce2ac903d.PNG)

**Comment by dimsemenov** [comment]
Do you test on iOS9? I can't repeat for some reason on it.

**Comment by hudochenkov** [comment]
Yes, I've tested on iOS 9.0.2 and iOS 8 (don't know specific version), using last Chrome. Try to drag picture up so it should be approximately at position on the screenshot.

I think while dragging picture finger is positioned on the address bar and Chrome not registered touchEnd.
